custom_connectors/custom_auth/kizen.rb

```diff
@@ -632,8 +858,312 @@
         get("https://api.kizen.com/#{input['object']}?page=1&page_size=1")
           .dig('results', 0)
       end
-    }
+    },
+    log_interaction: { # This is new
```

Comment:
Can we group `log activity` and `log interaction` together? into `log event` action. This can be easily done by following the patterns in the `create_object` and `update_object` steps

Reply:
I don't really want to do this. It is important to have all possible actions shown to the end user on the very first screen(https://www.workato.com/integrations/kizen) so they can see how many actions and triggers are possible.